contracts/abigen-with-interfaces.Dockerfile

```diff
@@ -0,0 +1,17 @@
+# use the below command to build this docker image locally
```

Comment:
I am not against this idea but who's going to maintain your branch with abigen updates?  like what if there are fixes and upgrades to abigen which we want to incorporate?

Reply:
I'll have to maintain it. that's just the nature of software. Pick your poison: either I maintain this or I have to maintain writing interfaces by hand and updating them much more frequently.

I have a PR open on geth to merge these changes: https://github.com/ethereum/go-ethereum/issues/28278

Reply:
as long as we are good with maintaining it - we are good.